src/app.ts

```diff
@@ -43,18 +43,22 @@ import {
 } from './utils';
 
 const FSH_VERSION = '2.0.0';
+const SUPPORTED_COMMANDS = ['build', 'updateDependencies', 'init', '-h', '--help'];
```

Comment:
This list needs to include `'help'` as well. This lets you do `sushi help build` to get the help text for a specific command.

Reply:
Updated in [fe15249](https://github.com/FHIR/sushi/pull/1163/commits/fe15249a27e11d26faf9284e0d4a048f55960078).